### DISCUSSION TITLE
Any guide or write up on understanding how the json data is read?

### DISCUSSION THREAD

**Discussion by stardustcafe** [danwild/leaflet-velocity#82] [Q&A]
Hello first of all amazing work. Secondly I wish to know if there is guide to understand the json data which is created. I want to implement this for our own data and the output file has 2d wind grid. However data in json seems 1D. 
I only need to know how to interpret the la1, la2 and other parameters so that I can implement same for our own website.

Thank you

**Comment by brentfraser** [comment]
The json data is created by https://github.com/danwild/wind-js-server, which uses the grib2json utility (https://github.com/cambecc/grib2json).  The json objects are a reflection of the original grib2 files, documented at https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/

**Comment by Pratik-Hub** [comment]
as brenfraser said we have to use ```[grib2json](https://github.com/cambecc/grib2json)```. grib2json convert the .f**** files for example .f000 or .f030 to a valid json file. For  example in your terminal run ```grib2json -d -n -o <your-output-file-name>.json <your input file name>``` and then you will get the json data which is valid. You can get data from [NCEP](https://nomads.ncep.noaa.gov/cgi-bin/filter_gfs_1p00.pl)
